apps/look-and-feel-stories/src/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Toggle } from "@axa-fr/design-system-apollo-react/lf";
```

Comment:
pour le moment, il faut aller chercher le package L&F
```suggestion
import { Toggle } from "@axa-fr/design-system-look-and-feel-react";
```